locallib.php

```diff
@@ -1138,7 +1140,7 @@ function moodleoverflow_print_post($post, $discussion, $moodleoverflow, $cm, $co
                                    $footer = '', $highlight = '', $postisread = null,
                                    $dummyifcantsee = true, $istracked = false,
                                    $iscomment = false, $usermapping = [], $level = 0,
-                                   $multiplemarks = false, $limitedanswertime = 0) {
+                                   $multiplemarks = false, $limitedanswersetting = 0) {
```

Comment:
The parameter is an std Class and should be null and not "0". Replace with ?$limitedanswersetting = null